lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -2423,9 +2423,13 @@ void PartitionCloner::visitCondBranchInst(CondBranchInst *inst) {
     assert(eltTy->isBuiltinIntegerType(1) && "expected Tensor<i1>");
 
     auto name = ctx.getIdentifier("tf_tensor_to_i1");
+    SmallVector<GraphOperationAttribute, 1> attributes;
```

Comment:
is it worth refactoring this and the next call-site into calling createTFInt1ToBuiltinInt1(), and perhaps moving that to TFUtilities?

Reply:
Done.